### PR TITLE
Droth 663 when a linear asset is expired mvalues are updated

### PR DESCRIPTION
**Pull request by MAC-CGI**
Droth-663 when a linear asset is expired mvalues are updated

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/oracle/OracleLinearAssetDao.scala [hunk 1]
```diff
@@ -1115,6 +1115,12 @@ class OracleLinearAssetDao(val vvhClient: VVHClient) {
     }
 
     insertProhibitionValue(id, value)
+    (startMeasure, endMeasure) match {
+      case (None, None) => None
```
Comment (joanacoelho) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/oracle/OracleLinearAssetDao.scala]: try this : 
```
case  (Some(start), Some(end)) => updateMValues(id, (start, end))
case _ => None
```

**Comment by halied** [comment]
Ok

**Comment by joanacoelho** [comment]
Ok for me

**Comment by joanacoelho** [comment]
ok for me

**Comment by marquesfm** [comment]
ok for me

**Pull request merged**